scripts/asm-installer/tests/common.sh

```diff
@@ -710,7 +714,8 @@ create_workload_service_account() {
 }
 
 create_new_instance_template() {
-  INSTANCE_TEMPLATE_NAME="vm-${LT_NAMESPACE}"
+  local SOURCE_INSTANCE_TEMPLATE="$1"
+  INSTANCE_TEMPLATE_NAME="$2"
```

Comment:
Similar to my previous comment, this can cause issues. If you take parameters in a function, the value should be assigned to a local variable, otherwise you could modify the global variable unexpectedly and these global variables can be used by other functions.

Reply:
you mean I make the INSTANCE_TEMPLATE_NAME to a local variable?